dotCMS/src/integration-test/java/com/dotcms/workflow/helper/TestWorkflowHelper.java

```diff
@@ -0,0 +1,73 @@
+package com.dotcms.workflow.helper;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.contenttype.model.type.BaseContentType;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.datagen.ContentTypeDataGen;
+import com.dotcms.datagen.ContentletDataGen;
+import com.dotcms.datagen.FieldDataGen;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.contentlet.model.ContentletDependencies;
+import com.dotmarketing.portlets.contentlet.model.IndexPolicy;
+import com.dotmarketing.portlets.workflows.business.SystemWorkflowConstants;
+import com.dotmarketing.portlets.workflows.business.WorkflowAPI;
+import com.liferay.portal.model.User;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Test {@link WorkflowHelper}
+ * @author jsanca
+ */
+public class TestWorkflowHelper extends IntegrationTestBase {
+
+    private static User user;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+        user = APILocator.getUserAPI().getSystemUser();
+    }
+
+    /**
+     * Test the getActionIdByName
+     */
+    @Test
+    public void test_getActionIdByName () throws Exception {
+
+        //1 create a content type and associated to system workflow
+        final ContentType contentGenericType = new ContentTypeDataGen().workflowId(SystemWorkflowConstants.SYSTEM_WORKFLOW_ID)
+                .baseContentType(BaseContentType.CONTENT)
+                .field(new FieldDataGen().name("title").velocityVarName("title").next())
+                .field(new FieldDataGen().name("body").velocityVarName("body").next()).nextPersisted();
+
+        final String unicodeText = "Numéro de téléphone";
+        final ContentletDataGen contentletDataGen = new ContentletDataGen(contentGenericType.id());
+        Contentlet contentlet    = contentletDataGen.setProperty("title", "TestContent")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'contentlet' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4801230)

Reply:
Done